test/Microsoft.ServiceHub.Framework.Tests/ServiceBrokerAggregatorTests.cs

```diff
@@ -192,7 +198,7 @@ public async Task ForceMarshal_GetProxyAsync()
 		ICalculator? result = await aggregator.GetProxyAsync<ICalculator>(Descriptor1, this.TimeoutToken);
 		Assumes.NotNull(result);
 		Assert.NotSame(result, this.mockBrokers[1].PipeResults[Descriptor1.Moniker]);
-		Assert.Equal(8, await result.AddAsync(3, 5));
```

Comment:
Why did you need to remove the AddAsync line?

Reply:
The MockPipe returned from GetPipeAsync method of MockBroker in these tests don't actually do anything. I assumed that's why GetPipeAsync tests wasn't calling any methods either.